xml/Microsoft.Extensions.Hosting.WindowsServices/WindowsServiceLifetime.xml

```diff
@@ -175,9 +175,9 @@
         <Parameter Name="cancellationToken" Type="System.Threading.CancellationToken" />
       </Parameters>
       <Docs>
-        <param name="cancellationToken">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="cancellationToken">Used to indicate when stop should no longer be graceful.</param>
+        <summary>Called at the start of <see cref="M:Microsoft.Extensions.Hosting.IHost.StartAsync(System.Threading.CancellationToken)" /> which will wait until it's complete before continuing. This can be used to delay startup until signaled by an external event.</summary>
+        <returns>A <see cref="T:System.Threading.Tasks.Task" />.</returns>
```

Comment:
```suggestion
        <param name="cancellationToken">A cancellation token that indicates when stop should no longer be graceful.</param>
        <summary>Asynchronously waits until start is complete before continuing. This method is called at the beginning of <see cref="M:Microsoft.Extensions.Hosting.IHost.StartAsync(System.Threading.CancellationToken)" />. This can be used to delay startup until signaled by an external event.</summary>
        <returns>A task that represents the asynchronous waiting for start operation.</returns>
```